seed/models/columns.py

```diff
@@ -583,6 +583,9 @@ class Column(models.Model):
     import_file = models.ForeignKey('data_importer.ImportFile', blank=True, null=True)
     units_pint = models.CharField(max_length=64, blank=True, null=True)
 
+    # 0 is deactivated. Order used to construct full address.
+    geocoding_order = models.IntegerField(default=0, blank=False)
```

Comment:
I'm excited to see that we are starting to more thoroughly use columns! There was a time not too long ago where columns were an abstract idea :)